packages/l/libmem/xmake.lua

```diff
@@ -0,0 +1,39 @@
+package("libmem")
+
+    set_homepage("https://github.com/rdbo/libmem")
+    set_description("Cross-platform game hacking library for C, C++, Rust, and Python, supporting process/memory hacking, hooking, detouring, and DLL/SO injection.")
+    set_license("AGPL-3.0")
+
+    add_urls("https://github.com/rdbo/libmem.git", {submodules = true})
+    --add_urls("https://github.com/rdbo/libmem/archive/refs/tags/$(version).tar.gz", {submodules = true}) 
+    add_versions("5.0.2", "99adea3e86bd3b83985dce9076adda16968646ebd9d9316c9f57e6854aeeab9c")
+    add_deps("cmake", "vcpkg::capstone", "vcpkg::keystone") -- "vcpkg::llvm 11.1.0"
+    on_install(function (package)
+        local configs = {
+            build_tests = package:config("build_tests"),
+            deep_tests = package:config("deep_tests"),
+            build_static = package:config("build_static")
```

Comment:
we need not tests. please always set it as false.

Reply:
and you did not use `add_configs` to add these configs.

Reply:
`package:config("shared")` is bultin config, use it instead of `build_static`

Reply:
Done I use shared now.

Reply:
```diff
-    if has_config("build_static") then
-        set_kind("static")
-    else
-        set_kind("shared")
-    end
+    set_kind("$(kind)")
 
     add_files(libmem_src)
-    
+
```
Maybe should be used.

Reply:
Done.